panther_description/urdf/gazebo.urdf.xacro

```diff
@@ -69,9 +74,17 @@
   </xacro:macro>
 
   <!-- IMU specification: https://www.phidgets.com/?tier=3&catid=10&pcid=8&prodid=1025#Tab_Specifications -->
-  <xacro:macro name="imu" params="reference_frame name:='imu'">
+  <xacro:macro name="imu" params="reference_frame namespace:=''">
+
+    <xacro:if value="${namespace == ''}">
+      <xacro:property name="ns" value="" />
```

Comment:
I'm thinking how we could change `ns` variable name because `ns` is not readable. Maybe `prefix` or `ns_prefix`?